include/lsst/daf/base/PropertySet.h

```diff
@@ -156,6 +156,13 @@ class LSST_EXPORT PropertySet {
      */
     bool isUndefined(std::string const& name) const;
 
+    /**
+     * Get the number of values in the entire PropertySet, including vectors.
```

Comment:
Could you please clarify this slightly? What does it do with vectors? Count a vector as 1 or is the number of elements added to the total.